media/js/wiki.js

```diff
@@ -847,5 +847,73 @@
     if($('details').length){
         initDetailsTags();
     }
+    
+    function create_jsfiddle(title, htmlCode, cssCode, jsCode) {
+       var $form = $('<form method="post" target="_blank" action="https://jsfiddle.net/api/post/library/pure/">'
+            + '<input type="hidden" id="html" name="html" />'
```

Comment:
I thought `id` would be better performance-wise, although in this case since we have very few elements - it shouldn't make much difference.